src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -110,7 +114,7 @@ public override void Initialize([AllowNull] ViewWithData data)
                 .ObserveOn(RxApp.MainThreadScheduler)
                 .Catch<System.Reactive.Unit, Octokit.AuthorizationException>(ex =>
                 {
-                    // TODO: Do some decent logging here
+                    log.Error(ex, "Error Processing PullRequests");
```

Comment:
This is explicitly an AuthorizationException not a general one, probably should have a better message i.e.
"Authorization error listing pull requests"